tools/depends/native/libjpeg-turbo/Makefile

```diff
@@ -24,14 +19,14 @@ $(TARBALLS_LOCATION)/$(ARCHIVE):
 $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
-	cd $(PLATFORM); $(CONFIGURE)
+	cd $(PLATFORM); patch -p1 -i ../01-disable-executables.patch
+	cd $(PLATFORM); $(CMAKE) -B build -DCMAKE_TOOLCHAIN_FILE= -DCMAKE_ASM_NASM_COMPILER:FILEPATH=$(NATIVEPREFIX)/bin/yasm -DENABLE_SHARED:BOOL=OFF -DWITH_JPEG8:BOOL=ON
```

Comment:
for the DCMAKE_ASM_NASM_COMPILER is it worth using the existing NASM var?

https://github.com/xbmc/xbmc/blob/353d61085acd8af33394051d207a5eabdd2fd657/tools/depends/native/config.site.native.in#L26

also, purely for reference if anyone looks in the future, libjpeg-turbo build docs say it can use either yasm or nasm, with some caveats around macos nasm min version apparently

> If using NASM, 2.10 or later is required.
If using NASM, 2.10 or later (except 2.11.08) is required for an x86-64 Mac build (2.11.08 does not work properly with libjpeg-turbo's x86-64 SIMD code when building macho64 objects.)
If using YASM, 1.2.0 or later is required.
If building on macOS, NASM or YASM can be obtained from MacPorts or Homebrew.
NOTE: Currently, if it is desirable to hide the SIMD function symbols in Mac executables or shared libraries that statically link with libjpeg-turbo, then NASM 2.14 or later or YASM must be used when building libjpeg-turbo.

https://github.com/libjpeg-turbo/libjpeg-turbo/blob/master/BUILDING.md